### ISSUE TITLE
Calibre run WordDumb with no errors but book has not any features (x-ray and wordwise)

### ISSUE THREAD

**Issue by miraculusik** [xxyzz/WordDumb#111]
### Checkboxes

- [X] I have read the document at [xxyzz.github.com/WordDumb](https://xxyzz.github.com/WordDumb).
- [X] I have not found similar issue or disscussion at GitHub.
- [X] Reboot doesn't fix the problem.

### Describe the bug

Calibre run WordDumb with no errors but book has not any features (x-ray and word wise). 
I tried azw3 and kfx format for WordDumb however it doesn't work. Again i don't get any error message. 
After applied WordDumb on my book. I upload books to my kindle via usb connection. And i open the book on my kindle, X-ray options appears but not activate and i can't active because it's not clickable. 

Option of word wise is always open but i don't get any word-wise explanation. I tried word wise close and open again but it doesn't work.

In short, I successfully convert, but my book does not change or add feature.

### System Information

OS: Macos Ventura (arm64)
Reader: Kindle Paperwhite 5 (11th gen)
calibre version: 6.14.0
Python version: 3.10 & 3.11
Plugin version: 3.29.0

### Error message

```text
None
```

### Reproduce steps

1. Convert Book `kfx` or `azw3` format
2. Select book and click WordDumb icon
3. Process complete successfully
4. Send to kindle via usb cable
5. Open the book on kindle ( X-ray and Word wise didn't work)

**Comment by xxyzz** [comment]
How did you send the book to Kindle? Word Wise and X-Ray files can only be sent automatically to device by clicking the WordDumb plugin icon or menu.

**Comment by miraculusik** [comment]
> How did you send the book to Kindle? Word Wise and X-Ray files can only be sent automatically to device by clicking the WordDumb plugin icon or menu.

in `Reproduce Steps`
After generated book with WordDumb. I click `send to device` option and i select format to send the book.

**Comment by miraculusik** [comment]
If I'm not mistaken. Do I first send the book to my kindle and then click worddumb for generate X-ray and Word-wise?
Because WordDumb doesn't automatically send book to my kindle.

**Comment by miraculusik** [comment]
[in this link](https://www.reddit.com/r/Calibre/comments/ml09rh/comment/izpz4vi/?utm_source=share&utm_medium=web2x&context=3) i found my mistake. These steps worked for me.

**Comment by xxyzz** [comment]
You only have to click the plugin once with your device connected to calibre and the plugin will send the book and other files. There is a video tutorial in the usage document: https://xxyzz.github.io/WordDumb/usage.html#create-files

**Comment by CottonCandyJuice** [comment]
I am not an expert but had the same problem, tried 1 hour and finally solve it. It is basically deleting the old Worddumb files from your computer, and then generating them again when it is connected to your device. 

Here is what I did.

1. Connect the device to your mac and remove the book from my kindle using Calibre. 

2. Search for the name of the book on my mac finder, found a few folders saved under the Calibre shelf, and deleted the other files named "languagelayer", "XRAY" and "BOOKNAME.apnx".  (I believe these files are the Worddumb files)

3. Now generate the Worddumb again. Both the book and the Worddumb should be automatically sent to your kindle without the need of clicking "send to device" manually.

Hope it helps.

**Comment by xxyzz** [comment]
You don't have to delete the created files and the book, just connect your Kindle then click the plugin icon or menu and the files will be moved to Kindle.
